src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/FileBinding.java

```diff
@@ -97,6 +98,7 @@ protected void copy(FilePath secret, FileCredentials credentials) throws IOExcep
         secret.copyFrom(credentials.getContent());
     }
 
+    @Symbol("file")
```

Comment:
I think 🐛 - as with the others for namespace collision with other things - probably fileCredential or fileSecret is much easier to understand than just 'file'